sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -79,7 +79,6 @@ export interface AmqpMessageProperties {
     replyToGroupId?: string;
     subject?: string;
     to?: string;
-    userId?: string;
```

Comment:
service-bus would also require the update

Reply:
Done.